src/Nethermind/Nethermind.Blockchain.Test/TransactionSelectorTests.cs

```diff
@@ -132,7 +133,7 @@ void SetAccountStates(IEnumerable<Address> missingAddresses)
                 }
 
                 stateProvider.Commit(Homestead.Instance);
-                stateProvider.CommitTree();
+                stateProvider.CommitTree(0);
```

Comment:
I would like to remove the block number from here and move the TrieSTore commit outside and use TrieNodeCache internally to PatriciaTrie